### PR TITLE
Home page

### LINKED ISSUE #25: Home Page
image

### PR DESCRIPTION
**Pull request by mustafaguldag**
Hi , I made the home page container. I took the components we need and also pass them.  #25 
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 import "./App.css";
 import "bootstrap/dist/css/bootstrap.min.css";
 
 function App() {
+
+
+
+
+
   return (
```
Comment (Rofy87) [inline src/App.js:9]: hi @mustafaguldag  Extra spaces

File: src/pages/homepage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const dataOfHeroSection = {
+  img: `${milkdog}`,
+  title: "Adopt Please",
+  text: "We are non-profit organization who wants to help people who wants to adopt",
+  side: "left",
+};
+
+function Homepage() {
+
```
Comment (Rofy87) [inline src/pages/homepage/HomePage.js]: Hi @mustafaguldag  extra spaces

File: src/App.js [hunk 3]
```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import Homepage from "./pages/homepage";
+import Homepage from "./pages/homepage/HomePage";
 import NavBar from "./components/NavBar/NavBar";
 import "bootstrap/dist/css/bootstrap.min.css";
 import "./App.css";
 import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (Rofy87) [inline src/App.js]: css was added twice

**Review by Rofy87 — COMMENTED** [review]
Hi @mustafaguldag  you need to run yarn fix too the tests did not pass

**Review by halitbatur — APPROVED** [review]

**Review by ayse8888 — APPROVED** [review]
I see that the spaces and double css was removed

**Pull request merged**